conf/application.conf

```diff
@@ -170,7 +170,6 @@ contact-frontend {
 
 features {
   email-journey = true
-  vat = true
   sa = true
   cr111 = true
```

Comment:
can't we get rid of this one now as well?

Reply:
Duh... missed it 👍🏻